test/basic.html

```diff
@@ -492,6 +503,48 @@
         expect(tooltip._removeListeners.callCount).to.be.equal(1);
         expect(tooltip._addListeners.callCount).to.be.equal(1);
       });
+
+      test('offsetParent new spec behavior', function() {
+        var f = fixture('offset-parent-new-behavior');
+        var target = f.querySelector('#target');
+        var tooltip = f.querySelector('paper-tooltip');
+
+        var shadowRoot = f.attachShadow({mode: 'open'});
+        shadowRoot.innerHTML = `
+          <style>
+            .dialog {
+              display: block;
+              position: absolute;
+              left: 0px;
+              right: 0px;
+              width: 100px;
+              margin: auto;
+            }
+          </style>
+          <div class=dialog>
+            <slot></slot>
+          </div>
+        `;
+
+        var actualTooltip = dom(tooltip.root).querySelector('#tooltip');
+        assert.isTrue(isHidden(actualTooltip));
+
+        MockInteractions.focus(target);
+        assert.isFalse(isHidden(actualTooltip));
+
+        debugger;
+
+        var divRect = target.getBoundingClientRect();
+        expectToBasicallyEqual(divRect.width, 100);
+        expectToBasicallyEqual(divRect.height, 100);
+
+        var contentRect = tooltip.getBoundingClientRect();
+        expectToBasicallyEqual(contentRect.width, 70);
+        expectToBasicallyEqual(contentRect.height, 30);
+
+        expectToBasicallyEqual(contentRect.left, 415);
+        expectToBasicallyEqual(contentRect.top, 114);
```

Comment:
`left` here is dependent on the browser's window size.

```suggestion
        // Check for correct 'bottom' positioning: the horizontal center of the
        // target and tooltip are aligned, and the top of the tooltip is
        // `offset` pixels below the bottom of the target.
        expectToBasicallyEqual(
            contentRect.left + contentRect.width / 2,
            divRect.left + divRect.width / 2);
        expectToBasicallyEqual(contentRect.top - divRect.bottom, tooltip.offset);
```